Util_AttachToActions.php

```diff
@@ -131,6 +131,18 @@ public function on_comment_change( $comment_id ) {
 		$this->on_post_change( $post_id );
 	}
 
+	/**
+	 * Comment status action
```

Comment:
What does "Comment status action" mean?

Reply:
WordPress features quick-links under comment items on the Comments admin page that allow administrators to alter the status of a comment with a single link. IE approve/spam/trash. These don't appear to trigger the edit hook and instead trigger the wp_set_comment_status hook